apps/system/index.html

```diff
@@ -1019,7 +1019,7 @@ <h1 data-l10n-id="data-roaming-enabled-title"></h1>
         <img id="toaster-icon" role="presentation" class="toaster-icon"/>
         <div id="notification-container">
           <div id="toaster-title" class="toaster-title" dir="auto"></div>
-          <div id="toaster-detail" class="toaster-detail"></div>
+          <div class="toaster-detail"><bdi id="toaster-detail"></bdi></div>
```

Comment:
it seems that you should be able to just set dir="auto" on the <div>, which does the same thing and doesn't require additional markup (you will not need the <bdi> tag then)